src/screens/Login/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import {
+  HelpBlock,
+  FormGroup,
+  FormControl,
+  ControlLabel
+} from "react-bootstrap";
+import { Auth } from 'aws-amplify';
+import LoaderButton from '../../components/LoaderButton';
+import "./index.css";
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: false,
+      email: "",
+      password: "",
+      loggedInUser: null
+    };
+  }
+
+  validateForm() {
+    return (
+      this.state.email.length > 0 &&
+      this.state.password.length > 0
+    );
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.id]: event.target.value
+    });
+  }
+
+  handleSubmit = async event => {
+    event.preventDefault();
+
+    this.setState({ isLoading: true });
+
+    try {
+      const loggedInUser = await Auth.signIn({
+        username: this.state.email,
+        password: this.state.password
+      });
+
+      this.setState({
```

Comment:
Can also use shorthand `this.setState({ loggedInUser })`

Reply:
Use shorthand version.